MinuteRaid/UI/SunMoon.gd

```diff
@@ -0,0 +1,19 @@
+extends TextureRect
+
+var _elapsed = 0
+
+func _ready():
+	pass # Replace with function body.
+
+func _process(delta):
+	# Want to transition from day to night every minute
+	_elapsed += delta
+	_elapsed = fmod(_elapsed,120.0)
```

Comment:
I've seen this in two places now. Is it something to extract, or is there some context involved where we can't?

Reply:
if I use this pattern again I'll DRY it up - for now it feels a bit like it would be more work than it's worth to DRY up since it would require extracting into a script, etc.

Reply:
Yea exactly, only if it is the right thing and doesn't complicate things.